pyEpiabm/pyEpiabm/routine/simulation.py

```diff
@@ -116,6 +131,51 @@ def configure(self,
             folder, filename,
             output_titles)
 
+        self.status_output = False
+        self.infectiousness_output = False
+        self.ih_status_writer = None
+        self.ih_infectiousness_writer = None
+
+        if ih_file_params:
+            # Setting up writer for infection history for each person. If the
+            # ih_file_params dict is empty, then we do not need to record this
+            self.status_output = ih_file_params["status_output"] \
+                if "status_output" in ih_file_params else False
+
+            self.infectiousness_output = \
+                ih_file_params["infectiousness_output"] \
+                if "infectiousness_output" in ih_file_params else False
+            person_ids = []
+            for cell in population.cells:
+                person_ids += [person.id for person in cell.persons]
```

Comment:
```suggestion
            person_ids += [person.id for person in cell.persons for cell in 
                           population.cells]
```

Excuse the formatting, but just so you know, double list comprehensions are allowed (and used elsewhere in the code

Reply:
Actually, this doesn't seem to work - causes an error in the code

Reply:
Fixed the syntax - the tests will pass after the next push